File: LV1/이상한 문자 만들기.js
function solution(s) {
  return s
    .split(" ")
    .map((word) => {
      return word
        .split("")
        .map((char, j) => {
          return j % 2 === 0 ? char.toUpperCase() : char.toLowerCase();
        })
        .join("");
    })
    .join(" ");
}

File: LV1/이상한 문자 만들기.java
public class Solution {

  public String solution(String s) {
    String[] words = s.split(" ", -1);
    StringBuilder answer = new StringBuilder();

    for (int i = 0; i < words.length; i++) {
      String word = words[i];

      for (int j = 0; j < word.length(); j++) {
        char c = word.charAt(j);
        if (j % 2 == 0) {
          answer.append(Character.toUpperCase(c));
        } else {
          answer.append(Character.toLowerCase(c));
        }
      }

      if (i < words.length - 1) {
        answer.append(" ");
      }
    }

    return answer.toString();
  }
}
